github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/GitHubPRJobDslExtension.java

```diff
@@ -44,8 +45,20 @@ public Object commitStatusOnGH(Runnable closure) {
                 context.unstableAs(),
                 null,
                 null,
-                null
-        );
+                null);
```

Comment:
please don't move closing bracket. Keeping it under opened allows eyes scroll it easier (not looking where end of arguments/initialisation.